lib/plugins/xdrip-js.js

```diff
@@ -0,0 +1,330 @@
+'use strict';
+
+var _ = require('lodash');
+var moment = require('moment');
+var times = require('../times');
+var levels = require('../levels');
+
+function init(ctx) {
+  var utils = require('../utils')(ctx);
+  var firstPrefs = true;
+  var lastStateNotification = null;
+
+  var sensorState = {
+    name: 'xdrip-js'
+    , label: 'Sensor Status'
```

Comment:
Just a suggestion ... should we consider renaming this to something more broad like "CGM Status"?

Reply:
Sounds good ...
Is ```cgm``` already used